### ISSUE TITLE
Error: SMF_SOFTWARE_YEAR is 2019, should be 2020.

### ISSUE THREAD

**Issue by m4z** [SimpleMachines/SMF#5910]
#### Description
In my local GitLab replica of this repo's Travis checks, today I'm getting:
```
23 $ if find . -name "*.php" ! -path "./vendor/*" -exec php -l {} 2>&1 \; | grep "syntax error, unexpected"; then exit 1; fi
24 $ if php other/buildTools/check-version.php | grep "Error:"; then exit 1; fi
25 Error: SMF_SOFTWARE_YEAR is 2019, should be 2020.
29 ERROR: Job failed: exit status 1
```
I expect this will happen here too, on the next commit. :wink: 

### Steps to reproduce
1. Do a commit in 2020
2. See `git show -s --format=%ci HEAD` ([check-version.php#L48](https://github.com/SimpleMachines/BuildTools/blob/master/check-version.php#L48)) explode

### Environment (complete as necessary)
- Version/Git revision: future :robot:

**Comment by m4z** [comment]
When trying to fix this locally, I noticed it also has to be patched in the `BuildTools` repo files. Since that is checked out by Travis/GitLab on the fly, I can't fix it locally.

You'll have to do at least the following (I hope there's a tool for that somewhere):
```
find . -iname '*.php' | xargs -n 1 sed -i -e "s/define('SMF_SOFTWARE_YEAR', '2019');/define
('SMF_SOFTWARE_YEAR', '2020');/"
find . -iname '*.php' | xargs -n 1 sed -i -e "s/copyright 2019/copyright 2020/"
```

The latter probably also needs to be done for `*.js`.

**Comment by Sesquipedalian** [comment]
Fixed in #5912

**Comment by Sesquipedalian** [comment]
Incidentally, I am glad to see that check-version.php is working as intended, reminding us to update the year right away.
